src/Styles.css

```diff
@@ -0,0 +1,8 @@
+.Restaurants-list{
```

Comment:
extra minor:
no reason for it not to work with capital case -  most of our css selectors are lower case, as is the file name `style.css`